### ISSUE TITLE
Exo-View Hand Key Point Annotations

### ISSUE THREAD

**Issue by Dou11112** [facebookresearch/Ego4d#335]
Hello, 

I have a question regarding 2d hand key point annotations. The dataset has 2D hand key point annotations on the exo-views, even when the hands are completely occluded.

Because there is no separate section dedicated to this part in the paper---https://arxiv.org/pdf/2311.18259, this part is a bit unclear to me, so could you please enlighten me on this?

Based on the appendix 13.D.2. 3D human hand pose from ego-video, I am guessing that, in the dataset, the 2D hand key points on the exo-views with the label "automatic" are purely estimated by off-the-shelf body pose estimation models. Thus, the annotations are there even when the hands are occluded. would it be correct?

Also, for the key points with the "manual" label, we are certain that they are at least visible and the annotators were able to annotate them, right?

Thank you so much for the clarification in advance!

**Comment by suyogduttjain** [comment]
Yes keypoints with tag "automatic" are projections from the 3D pose and thus can be present without a visible hand. Keypoints with "manual" tag are done by human annotators and you expect them to be only present when hands are visible. 

You can run a hand detector or apply some bio-mechanical constraints to filter out big outliers.

**Comment by SJTUwxz** [comment]
Closing this issue for now. Please feel free to re-open it if the problem persists. Thanks!
